packages/events/EventPluginUtils.js

```diff
@@ -78,12 +80,22 @@ if (__DEV__) {
 function executeDispatch(event, simulated, listener, inst) {
   const type = event.type || 'unknown-event';
   event.currentTarget = getNodeFromInstance(inst);
-  ReactErrorUtils.invokeGuardedCallbackAndCatchFirstError(
-    type,
-    listener,
-    undefined,
-    event,
-  );
+  if (type === 'change') {
```

Comment:
This seemed like the obvious place to put this check, but perhaps there's a higher level caller where this can go instead?

Reply:
Could you do it in `ChangeEventPlugin` instead?